silero_vad_iterator.py

```diff
@@ -9,10 +9,10 @@
 class VADIterator:
     def __init__(self,
                  model,
-                 threshold: float = 0.5,
-                 sampling_rate: int = 16000,
+                 threshold: float = 0.50,
+                 samplerate: int = 16000,
```

Comment:
thanks. One useful suggestion from this PR  is to rename various ways of sampling_rate / SAMPLINGRATE etc. to uniform way